### ISSUE TITLE
Output file format and naming convention

### ISSUE THREAD

**Issue by jtcramer** [marl/openl3#14]
I have some questions about how to deal with embedding outputs:

- Should we include the timestamps? If so do we save it in the same file?
- What format should we use?
    - h5: Nice compression options, but since these typically shouldn't be large, it might be more annoying to deal with than other options
    - npy/npz: Standard approach, can easily load numpy arrays directly
    - JAMS: Using JAMs would help expand its use and would have a natural way to associate the timestamps with each embedding, but storing all of the values as text might be cumbersome and make the files big, especially if they are long
- Should we use the embedding type to name the embedding? e.g. `example_audio_openl3_6144emb_linear_music.<ext>` Or should we just keep it simple?
    - It might be good if the user is comparing different embeddings, but it might be cumbersome if people just want to use a single type of embeddings. Of course we could add an option for this, but adding another option for something like this might be excessive.

**Comment by justinsalamon** [comment]
My 2c:
* By default the output can include 2 arrays, one with timestamps and another with the embeddings. We can add a `--no-timestamps` option if people want only the embedding as output.
* I'd probably go with npy for the default output format. Later if we see there's demand we can add `--h5-output` and `--jams-output` as options.
* We could provide an optional `--suffix` to allow the user to define their own desired suffix, and by default set it to the empty string.

Thoughts?

**Comment by jtcramer** [comment]
I'd say the options for those things might be nice, but maybe those things should be lower priority for now. Unless there are objections, let's just have a single output with both arrays, npy, and no suffix for now.

**Comment by justinsalamon** [comment]
I agree about initially only supporting npy output with both arrays. I'm less sure about not supporting the suffix, I think that would be very useful for users (at least for me), and implementing it should be a breeze.

**Comment by jtcramer** [comment]
It's been added to the API and the tests have been updated to reflect that.
